### DISCUSSION TITLE
any roadmap or planning for mono interpreter execute optimation?

### DISCUSSION THREAD

**Discussion by srxqds** [dotnet/runtime#110855] [General]
any roadmap or planning for mono interpreter execute optimation?

**Comment by srxqds** [comment]
I found dotnet9 have supported native aot for ios https://learn.microsoft.com/en-us/dotnet/maui/deployment/nativeaot?view=net-maui-9.0

but, we are worring the dotnet team maybe not continue to work on mono interpreter.

**Comment by huoyaoyuan** [comment]
There was interest to resecure the coreclr interpreter to be functional (https://github.com/dotnet/runtimelab/tree/feature/CoreclrInterpreter), but the initial experiment wasn't successful (https://github.com/dotnet/runtimelab/pull/2869 https://github.com/dotnet/runtimelab/pull/2872).

Though not officially announced, the future interest should be bringing the capabilities with coreclr and nativeaot.
